tests/foreman/installer/test_installer.py

```diff
@@ -1413,7 +1413,18 @@ def test_capsule_installation(sat_default_install, cap_ready_rhel, default_org):
         3. health check runs successfully
 
     :CaseImportance: Critical
+
+    :BZ: 1984400
+
+    :customerscenario: true
     """
+    # Setup HTTP Proxy to add coverage for BZ#1984400
+    http_proxy_setting = sat_default_install.api.Setting().search(
+        query={'search': 'HTTP(S) proxy'}
+    )[0]
+    http_proxy_setting.value = settings.http_proxy.un_auth_proxy_url
+    http_proxy_setting.update(['value'])
```

Comment:
I'd recommend using `update_setting` here and at teardown https://github.com/SatelliteQE/robottelo/blob/master/robottelo/hosts.py#L2012